includes/class-wcpdf-admin.php

```diff
@@ -552,20 +555,29 @@ public function get_current_values_for_document( $document, $data ) {
 			),
 		);
 
-		if ( !empty( $data['notes'] ) ) {
+		if ( ! empty( $data['notes'] ) ) {
 			$current['notes'] = array(
 				'value' => $document->get_document_notes(),
-				'name'  =>"_wcpdf_{$document->slug}_notes",
+				'name'  => "_wcpdf_{$document->slug}_notes",
 			);
 		}
 
-		if ( !empty( $data['display_date'] ) ) {
+		if ( ! empty( $data['display_date'] ) ) {
 			$current['display_date'] = array(
 				'value' => $document->document_display_date(),
-				'name'  =>"_wcpdf_{$document->slug}_display_date",
+				'name'  => "_wcpdf_{$document->slug}_display_date",
 			);
 		}
 
+		if ( ! empty( $data['creation_status'] ) ) {
+			$document_triggers = WPO_WCPDF()->main->get_document_triggers();
+			$creation_status   = $document->get_creation_status();			
```

Comment:
I would rename this to: `$trigger = $document->get_trigger()`

This then also has to be changed in the rest of the code.

Reply:
If you look at the document functions here https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/d6c5ae258ec0551845f0f6f104f96c26ef593566/includes/documents/abstract-wcpdf-order-document.php#L459-L477

They are more descriptive in their names, and since we are labeling this as `Invoice created via:`, I don't think we should name this `trigger`. But let me know.

Reply:
But we also call them triggers when getting a list of all possible triggers for creating the document:

```
$document_triggers = WPO_WCPDF()->main->get_document_triggers();
```

Would you prefer it if we called it something like `creation_trigger`?

Reply:
`creation_trigger` seems more appropriate indeed.